wavelength-plugins/profile.js

```diff
@@ -499,6 +499,20 @@ exports.commands = {
 		"/nature delete - Removes your Profile Nature.",
 	],
 
+	'!lastactive': true,
+	checkactivity: 'lastactive',
+	lastactive: function (target, room, user) {
+		if (!target) target = user.userid;
+		if (target.length > 18) return this.errorReply("Usernames cannot exceed 18 characters.");
+		if (!this.runBroadcast()) return;
+		let targetUser = Users.get(target);
+		if (!Users(targetUser) || !targetUser.connected) return this.errorReply(`${target} is not online. Use /seen to find out how long ago they left.`);
```

Comment:
`if (!Users(targetUser))` - seeing as `Users` is an alias of `Users.get`, you're essentially checking `Users(Users(target))`, which is redundant. I think you should be able to use `if (!targetUser || !targetUser.connected)`.

This isn't a change you have to make now; however...looking into TypeScripting everything properly, I noticed that `Users.get` returned very odd behaviour. It has to be done specifically in order to get everything to comply properly with TypeScript, so when I open a PR on it, this will have to be changed. I'll have to note specifically how to do it later.

Reply:
Okie

Reply:
imo i think the ``!Users(targetUser)`` should just be ``!targetUser`` so theres no possible errors later

Reply:
because like @Mystifi said it's redundant